System.IO.Abstractions/DirectoryInfoBase.cs

```diff
@@ -7,7 +7,7 @@ namespace System.IO.Abstractions
     [Serializable]
     public abstract class DirectoryInfoBase : FileSystemInfoBase
     {
-        protected DirectoryInfoBase(IFileSystem fileSystem) : base(fileSystem)
+        protected DirectoryInfoBase(IFileSystem fileSystem, string path) : base(fileSystem, path)
```

Comment:
To reduce public API changes and therefore downstream churn please undo the changes in this file and override `ToString` in `*Wrapper`/`Mock*` instead.

Reply:
ToString moved to MockDirectoryInfo